cypress/support/commands.js

```diff
@@ -23,3 +23,19 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+cypress.Commands.add('login', (username, password) => {
+  cy.request({
```

Comment:
Add a default value for 'username' and 'password' parameters in case they are not provided.